modules/ROOT/assets/attachments/sg.yaml

```
@@ -167,32 +167,50 @@ properties:
             properties:
               max_wait_pending:
                 type: integer
-                description: Maximum wait time in milliseconds for a pending sequence before skipping sequences.
+                description: (Deprecated) Maximum wait time in milliseconds for a pending sequence before skipping sequences.
                 default: 5000
               max_num_pending:
                 type: integer
-                description: Maximum number of pending sequences before skipping the sequence.
+                description: (Deprecated) Maximum number of pending sequences before skipping the sequence.
                 default: 10000
               max_wait_skipped:
                 type: integer
-                description: Maximum wait time in milliseconds for a skipped sequence before abandoning the sequence.
+                description: (Deprecated) Maximum wait time in milliseconds for a skipped sequence before abandoning the sequence.
                 default: 3600000
               enable_star_channel:
                 type: boolean
-                description: Enable the star (*) channel.
+                description: (Deprecated) Enable the star (*) channel.
                 default: 'true'
               channel_cache_max_length:
                 type: integer
-                description: Maximum number of entries maintained in cache per channel.
+                description: (Deprecated) Maximum number of entries maintained in cache per channel.
                 default: 500
               channel_cache_min_length:
                 type: integer
-                description: Minimum number of entries maintained in cache per channel.
+                description: (Deprecated) Minimum number of entries maintained in cache per channel.
                 default: 50
               channel_cache_expiry:
                 type: integer
-                description: Time (seconds) to keep entries in cache beyond the minimum retained.
+                description: (Deprecated) Time (seconds) to keep entries in cache beyond the minimum retained.
                 default: 60
+            channel_cache:
+              type: object
+              description: Channel cache configuration
+              properties:
```

Added high/lower watermark options not in the config tech spec:

```suggestion
              properties:
                compact_high_watermark_pct:
                  type: integer
                  description: High watermark for channel cache eviction (percent)
                  default: 80
                compact_low_watermark_pct:
                  type: integer
                  description: Low watermark for channel cache eviction (percent)
                  default: 60
```